src/testing/fixtures/algorand-fixture.ts

```diff
@@ -86,34 +72,36 @@ export function algorandFixture(fixtureConfig: AlgorandFixtureConfig | undefined
  * @param config The algo configuration
  * @returns The fixture
  */
+export function algorandFixture(fixtureConfig: AlgorandFixtureConfig | undefined, config: AlgoConfig): AlgorandFixture
+
 export function algorandFixture(fixtureConfig?: AlgorandFixtureConfig, config?: AlgoConfig): AlgorandFixture {
-  config = config || getConfigFromEnvOrDefaults()
+  fixtureConfig = { ...fixtureConfig, ...config }
+  if (!fixtureConfig.algod || !fixtureConfig.indexer || !fixtureConfig.kmd) {
+    fixtureConfig = { ...ClientManager.getConfigFromEnvironmentOrLocalNet(), ...fixtureConfig }
+  }
```

Comment:
I've changed the signature to allow config OR clients to be passed in via `AlgorandFixtureConfig` type now, much cleaner than a weird way of calling it where you add `undefined` to the first parameter and this code here is less confusing now too. I've marked the override as deprecated to reflect this change, that version will still work so it's not a breaking change.